packages/accordions/src/elements/timeline/components/Item.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useMemo,
+  forwardRef,
+  Children,
+  ReactNode,
+  ReactElement,
+  HTMLAttributes
+} from 'react';
+import { Timeline } from '../Timeline';
+import { StyledTimelineItem } from '../../../styled';
+import { TimelineItemContext, useTimelineContext } from '../../../utils';
+
+export interface IItem extends HTMLAttributes<HTMLLIElement> {
```

Comment:
Just curious if `IItem` is meant to just be `Item`?

Reply:
Garden adopted the convention to prefix interface names with the letter "I".

Reply:
Oh interesting, good to know 👍🏼